### PR TITLE
Simple/basic server with a GET endpoint

### LINKED ISSUE #6: Simple/basic server with a GET endpoint
Create a simple single-threaded server which returns 200 when doing a GET to /posts

### PR DESCRIPTION
**Pull request by vuong-buihv**
#6  Create a simple single-threaded server which returns 200 when doing a GET to /posts

### CODE REVIEW

**Review by nelup20 — CHANGES_REQUESTED** [review]
Small change (let to const), see comment diff view

File: src/main.rs [hunk 1]
```diff
@@ -1,15 +1,47 @@
...
+        println!("No .json file was provided as argument!");
+        process::exit(0);
+    }
+
+
+    let json_filename = &args[1];
+    let json_content = fs::read_to_string(json_filename).unwrap();
+
+
+    let ip = "127.0.0.1";
```
Comment (nelup20) [inline src/main.rs]: Looks good 👍, maybe we should change all these let's to const's?
At least for things like ip, port etc. --> also if we do that, name should be all uppercase. (IP, PORT).

Comment (vuong-buihv) [reply src/main.rs]: I was thinking about the possibility to put the ip and port as arguments later. But yea for the default values we may need constants or maybe put them in a config file. Not sure how to do it in rust.

Comment (nelup20) [reply src/main.rs]: Good point, yeah for those things we can have flags later, or maybe a special config file.
For now, I'm still thinking that we should start off with consts etc., maybe DEFAULT_IP and DEFAULT_PORT?

We can also define em outside of main but it's a matter of style/convention, tbd?

**Review by vuong-buihv — COMMENTED** [review]
Added a new commit defining IP and port as constants

**Review by nelup20 — APPROVED** [review]

**Pull request merged**